shot-consumer-compose/app/src/main/java/com/karumi/shotconsumercompose/ui/Theme.kt

```diff
@@ -0,0 +1,44 @@
+package com.karumi.shotconsumercompose.ui
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+
+private val DarkColorPalette = darkColors(
+        primary = purple200,
+        primaryVariant = purple700,
+        secondary = teal200
+)
+
+private val LightColorPalette = lightColors(
+        primary = purple500,
+        primaryVariant = purple700,
+        secondary = teal200
+
+        /* Other default colors to override
```

Comment:
Do we wanna keep this comment around?

Reply:
It's just the project android studio creates. I didn't want to modify the original code because we do not now how we are going to use it for now and I thought it could be easier for you to review.